paasta_tools/cli/cli.py

```diff
@@ -84,6 +84,10 @@ def get_argparser():
     )
 
     subparsers = parser.add_subparsers(help="[-h, --help] for subcommand help")
+    subparsers.required = True
+    # https://stackoverflow.com/questions/18282403/argparse-with-required-subcommands/18283730#18283730
+    subparsers.dest = 'command'
```

Comment:
This can be a keyword to `add_subparsers` (but `.required` cannot for some reason, maybe I'll make a PR to stdlib to enable this).  Here's an [example from pre-commit](https://github.com/pre-commit/pre-commit/blob/72efbb3950ef5943156c0171737b3d7edcfa00e7/pre_commit/main.py#L60) (note that pre-commit avoids needing `.required = true` due to some other hax that I wouldn't suggest (it implements a "default" command))